src/generators.jl

```diff
@@ -814,3 +814,91 @@ Usage
 `A = forest_fire_graph(c, k, p)` returns a forest-fire sample. 
 `A = forest_fire_graph(A0, k, p)` 
 """
+
+
+"""
+`partial_duplication`
+=====================
+
+A random graph model based off of the evolution of protein-protein interaction
+networks. The method takes an undirected graph and runs a number of steps 
+which randomly selects an existing vertex, duplicates the node, and keeps the
+edges between its neighbor with a given probability. In the original work 
+the authors prove that new vertices will tend to become isolated for 
+p <= 0.567143. 
+
+src: 
```

Comment:
I'm pretty sure this model predates 2014... seeing as I worked on it with Amin Saberi on 2004/2005... (and it wasn't new then...) see, e.g. https://en.wikipedia.org/wiki/Copying_network_models

Reply:
Also, 'src' is a little informal. The proper markdown is probably a definition list. https://www.markdownguide.org/extended-syntax/, eg.

```
Reference
: Large-scale behavior of the partial duplication random graph, 
  Felix Hermann & Peter Pfaffelhuber, <https://arxiv.org/pdf/1408.0904.pdf> 
```